tests/manage/noobaa/conftest.py

```diff
@@ -0,0 +1,15 @@
+import pytest
+
+from ocs_ci.ocs.resources import noobaa
+
+
+@pytest.fixture()
```

Comment:
Please move to conftest as discussed in a previous commit

Reply:
Another option would be to have the existing test case we have (bucket creation) and this new bucket deletion test case, in the same file

Reply:
I believe creating a conftest.py under NooBaa will be the best choice

Reply:
I had created conftest.py in noobaa dir in my previous commit.